src/en/clean-copy/03-Part II. The Globe of Belgium/15.md

```diff
@@ -1,65 +1,65 @@
 
-Coriander, on the contrary, could easily be an ingredient in Belgian beer as it wasn't an exotic spice at all. It had been being cultivated in Europe since at least the second millennium BCE and was a part of *gruit*. Then why hops in the recipe, though?
+Coriander, on the contrary, could easily have been an ingredient in Belgian beer as it wasn't an exotic spice at all. It had been being cultivated in Europe since at least the second millennium BCE and was a part of *gruit*. Gruit was a pre-modern herbal mixture for the bittering and balancing beer, predating stndardized hop use. Then what role *did* hops play in the recipe?
 
-Dutch beer of the 14th-16th centuries was at least half oats, which were then the main cereal. And it must have been dark, in the best case, amber, but definitely not pale. But there are neither oats nor dark malt in *witbier* — at least, in the ‘classical’ Hoegaarden one.
+Dutch beer of the 14th-16th centuries would have been at least half oats, which was then the main cereal grain. Aditionally, it would have been dark or in the best case, amber, but definitely not pale. However, there are neither oats nor dark malt in *witbier* — at least in the ‘classic’ Hoegaarden White.
 
-As for the sour taste, the late Medieval brewers were quite able to fight it without the use of coriander, as both chronicles and reconstructions demonstrate[ref:{"source":{"alias":"larsblog","short":"Garshol, L. M.","extra":"Larsblog","href":"https://www.garshol.priv.no"}}:"Was all beer sour before Pasteur?"](https://www.garshol.priv.no/blog/306.html), and orange zest could only increase acidity. 
+As for the souring of beer, late Medieval brewers were able to control or at least delay the souring of beer without the use of coriander, as both chronicles and reconstructions demonstrate[ref:{"source":{"alias":"larsblog","short":"Garshol, L. M.","extra":"Larsblog","href":"https://www.garshol.priv.no"}}:"Was all beer sour before Pasteur?"](https://www.garshol.priv.no/blog/306.html). Furhtermore, orange zest might only increase acidity. 
 
-Finally, what monks-experimenters we're talking about? Monasteries were obliged to be self-sustainable — implying growing oranges locally, which looks quite unrealistic in Belgian climate — to say nothing about cities and guilds, not monks, being the main drivers of innovation in brewing in the 15th-century Netherlands.
+Finally, what monks or experimentalosts are we talking about? Monasteries by their social make-up were obliged to be self-sustainable. This implies growing oranges locally which would be quite unrealistic in a Belgian climate. This does not even consider that it would have been city merchants and guilds, not monks, who would have been the main drivers of innovation in brewing within the 15th-century Netherlands.
```

Comment:
I changed it to "social organization." We can revisit when I've looked over the previous chapters.
I'm refering to their society, not their social habits. This will be understood.

Reply:
Ok